sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/ResourceExtensionsTests.cs

```diff
@@ -30,12 +31,20 @@ public void NullResource(string? instrumentationKey)
     [InlineData(InstrumentationKey)]
     public void DefaultResource(string? instrumentationKey)
     {
-        var resource = CreateTestResource();
-        var azMonResource = resource.CreateAzureMonitorResource(instrumentationKey);
+        try
+        {
+            Environment.SetEnvironmentVariable("EXPORT_METRIC_FOR_RESOURCE_ATTRIBUTES", "true");
```

Comment:
Shall we cover these cases where `EXPORT_METRIC_FOR_RESOURCE_ATTRIBUTES` set to null, empty.

Reply:
Add this to the last test theory within the current PR

Reply:
Done